hyperspy/_signals/eds_tem.py

```diff
@@ -910,7 +912,10 @@ def CL_get_mass_thickness(self, weight_percent, thickness):
             [elements_db[element]['Physical_properties']['density (g/cm^3)']
                     for element in elements])
         for density, element_composition in zip(densities, weight_percent):
-            elemental_mt = element_composition * thickness_map * 1E-9 * density
+            # convert composition from % to fraction: factor of 1E-2
+            # convert thickness from nm to m: factor of 1E-9
+            # convert density from g/cm3 to kg/m2: factor of 1E3
+            elemental_mt = element_composition * thickness_map * density * 1E-8
```

Comment:
Yes, this should be OK now

Reply:
I think the units of composition is fractional and not percentage.

Reply:
What make you think that this is fractional and not percentage?
According to the following, the result of the initial quantification is multiplied by 100 (in L449 below), so I would expect it to be percentage:
https://github.com/hyperspy/hyperspy/blob/c82ae3b1e01879bb0bb05dfc5685c00399f7465d/hyperspy/_signals/eds_tem.py#L446-L454

And `quantification_cliff_lorimer` returns weight fraction:
https://github.com/hyperspy/hyperspy/blob/c82ae3b1e01879bb0bb05dfc5685c00399f7465d/hyperspy/misc/eds/utils.py#L388-L410

Reply:
@ericpre sorry trying to multitask at the microscope this week and failing. You're right the composition at this point is in percent rather than fraction. I've done my best to comb through the rest of the code and I think it is all consistent apart from this one scaling error. I was looking in all the individual quantification functions and saw that they returned fractions and didn't check the umbrella function.

Reply:
No worries, this is easy to get lost in the conversion and this is why I pointed out where it came from exactly!